### ISSUE TITLE
org.openqa.selenium.safari.SafariDriver#toString fails with org.openqa.selenium.WebDriverException

### ISSUE THREAD

**Issue by graysr89** [SeleniumHQ/selenium#4525]
## Meta -
OS:  
MAC OS Sierra - 10.12.5 (16F73)

Selenium Version:  
3.5.1

Browser:  
Safari

Browser Version:  
 10.1.1

## Expected Behavior -
When one creates a new noargs SafariDriver and call toString() on it, the toString() should return a correct string representation of the driver.

## Actual Behavior -
When one creates a new noargs SafariDriver and calls toString() on it this fails with the exception 
```
org.openqa.selenium.WebDriverException: Unrecognized platform: macOS
Build info: version: '3.5.1', revision: '9c21bb67ef', time: '2017-08-17T15:26:08.955Z'
```

## Steps to reproduce -

```
import org.junit.Test;
import org.openqa.selenium.safari.SafariDriver;

public class SafariDriverTest {

    @Test
    public void testSafariDriverToString(){
        SafariDriver safariDriver = new SafariDriver();
        safariDriver.toString();
    }
}
```

**Comment by hypnoce** [comment]
Version 3.5.2 should fix it : https://github.com/SeleniumHQ/selenium/commit/0a4ccb9c679767803b053430005c4e98b9dd519f

**Comment by barancev** [comment]
Fixed in 3.5.2, yes.
